css/properties/animation.json

```diff
@@ -0,0 +1,149 @@
+{
+  "css": {
+    "properties": {
+      "animation": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/animation",
+          "support": {
+            "webview_android": {
```

Comment:
Unprefixed version supported.
Also I would remove the partial_implementation, the note is enough.